keel-core/src/main/kotlin/com/netflix/spinnaker/keel/api/Constraint.kt

```diff
@@ -64,7 +64,6 @@ data class ManualJudgementConstraint(
 ) : StatefulConstraint("manual-judgement")
 
 data class PipelineConstraint(
-  val serviceAccount: String = "anonymous",
```

Comment:
I took the service account off the constraint, as we can just get it from the top level now